src/background/adblocker.js

```diff
@@ -367,14 +373,7 @@ ${scripts.join('\n\n')}}
   );
 }
 
-async function injectScriptlets(tabId, frameId, url) {
-  try {
-    setup.pending && (await setup.pending);
```

Comment:
Not super sure how the runloop will handle that across different engines. 
But to be sure we are as fast as possible, ad-blocking path should always be sync. I've changed this place and all others to print a warning instead of an error as it is expected that from time to time adblocker wont startup in time.